uint8 Duty
uint8 Channel
---

